frame/ibc/src/lib.rs

```diff
@@ -243,6 +244,34 @@ pub mod pallet {
 	pub type HostConsensusStates<T: Config> =
 		StorageValue<_, BoundedBTreeMap<u64, IbcConsensusState, ConstU32<250>>, ValueQuery>;
 
+	// temporary
+	#[pallet::storage]
+	#[allow(clippy::disallowed_types)]
+	/// (ChannelId, PortId, Sequence) => Packet
+	pub type Packets<T: Config> = StorageDoubleMap<
+		_,
+		Blake2_128Concat,
```

Comment:
Why not `Blake2128Concat`?

Reply:
no such thing

Reply:
oh got it, comes from https://docs.rs/frame-support/latest/frame_support/struct.Blake2_128Concat.html
What an odd naming convention